test/e2e-encryption-kms/encryption_kms.go

```
@@ -72,19 +71,67 @@ func testKMSEncryptionProvidersMigration(ctx context.Context, t testing.TB) {
+		AssertResourceNotEncryptedFunc: library.AssertTokenOfLifeNotEncrypted,
+		ResourceFunc:                   library.TokenOfLife,
 		ResourceName:                   "TokenOfLife",
 		EncryptionProviders: library.ShuffleEncryptionProviders([]library.EncryptionProvider{
 			librarykms.DefaultVaultEncryptionProvider(ctx, t),
 			library.SupportedStaticEncryptionProviders[rand.IntN(len(library.SupportedStaticEncryptionProviders))],
 		}),
 	})
 }
+
+// testKMSToKMSMigration tests KMS-to-KMS migration (primary → secondary → primary → identity).
+func testKMSToKMSMigration(ctx context.Context, t testing.TB) {
```

golangci-lint (2.12.2)

[error] func testKMSToKMSMigration is unused

(unused)